### PR TITLE
Update amount of follow

### PR DESCRIPTION
**Pull request by dar2015**
Amount to follow for giving users follow split between each account.
For example if you have 10 different accounts you follow for followers intend of set for 20 each account you can set 200 for all and it will split it automatically.

### CODE REVIEW

**Comment by timgrossmann** [comment]
@dar2015 Thank you for contributing!

However, I don't really understand what this feature does, I'm sorry.
Could you explain it a little bit more?

**Comment by dar2015** [comment]
@timgrossmann today, when you set amount to follow it's will replicate the amount for each users you want to follow from his list. for example you set amount 100 and you have 5 different users you follow from the bot will follow 500 users. 
My update will split 100 following between all the 5 users and each list will follow over 20 users.

**Comment by dar2015** [comment]
Today
amount=100 
users=5
total follow = 500
5*100 = 500

My update
amount=100 
users=5
total follow = 100
100/5 = 20 * 5

**Comment by timgrossmann** [comment]
@dar2015 Ok now I get it.
But this most likely will only make sense if we can make it optional.
Could you think about making this a parameter or setting, please?

**Comment by dar2015** [comment]
I can do it, how you suggest to implement it?

**Comment by sionking** [comment]
I think this is great!
I always trim my list to 5 tags (out of 20 let say) to not excceed with total tags amount.
So total is preferred .

**Comment by GitNees** [comment]
Or put 20 with 5 users = 100 :D #quickmath

File: instapy/instapy.py [hunk 1]
```diff
@@ -1221,12 +1221,15 @@ def follow_user_followers(self,
         userFollowed = []
         if not isinstance(usernames, list):
             usernames = [usernames]
-        for user in usernames:
 
+        amount_per_user = int(ceil(amount / len(usernames)))
```
Comment (Tjorriemorrie) [inline instapy/instapy.py]: please handle zerodivisionerror. Add return above if empty
Does not seem necessary to cast it to int (scroll_bottom casts it)

**Review by Tjorriemorrie — CHANGES_REQUESTED** [review]

**Comment by Tjorriemorrie** [comment]
I'm not sure I understand this change. Just call the method with 20.

**Comment by dar2015** [comment]
Fixed for your request added as param

**Comment by GitNees** [comment]
Still a very confusing option. Its creating a solution for a problem that doesn't exist.

**Comment by sionking** [comment]
The problem:
The number of tags vary all the time, so if you have now 5 tags and amount 20 you will get **100**.
Tmrw you will add a new tag (6 tags) amount is the same and you will have the **120** instead.

Anyway I thought about it and still prefer to control the amount of tags I am entering.
A better feature will be amount per tag (use tuples). and total of 5 tags shuffled every run.

**Comment by Tjorriemorrie** [comment]
That's still an implementation detail.

**Comment by dar2015** [comment]
Idea of amount is to control how many follow do you want. I use external list of accounts that change for time to time and don't really know how many accounts on the list. the amount of follow changing for me now. this idea solve it.

**Comment by timgrossmann** [comment]
I'm extremely unsure about this...
We either put this as a parameter (if we don't overdue it) or keep it off...

**Comment by dar2015** [comment]
Now i able it as a parameter you can choose to by toggle it.

**Comment by Tjorriemorrie** [comment]
But you also pass in the usernames: so you have all the info for the calculation which you can do just before you call the function; and pass in the calculated amount.

**Pull request closed without merging**